pkgs/misc/preload/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchurl, autoconf, automake, pkg-config, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "preload";
+  version = "0.6.4";
+
+  src = fetchzip {
+    url =
+      "http://downloads.sourceforge.net/sourceforge/${pname}/${pname}-${version}.tar.gz";
```

```suggestion
      "mirror://sourceforge/preload/preload-${version}.tar.gz";
```

* uses SF mirror rather than using hardcoded http download link
* `pname` is semantically different from repo/project name